src/elements/content-sidebar/ContentSidebar.js

```diff
@@ -368,6 +370,7 @@ class ContentSidebar extends React.Component<Props, State> {
             onVersionHistoryClick,
             theme,
             versionsSidebarProps,
+            signSidebarProps,
```

Comment:
```suggestion
            signSidebarProps,
            theme,
            versionsSidebarProps,
```

Reply:
some other locations need some alphabetizing as well. e.g. (line 416 in this file, line 74, 316, 352 in `Sidebar.js`, etc.)